donation-api/src/main/java/org/interfaithsanctuary/donationapi/model/Donation.java

```diff
@@ -0,0 +1,71 @@
+package org.interfaithsanctuary.donationapi.model;
+
+import javax.persistence.*;
+import java.util.Date;
+
+/**
+ * This object describes the many-many relationship that a Donation can have with a Callout.
+ */
+@Entity
+@Table(name = "donations")
+public class Donation {
+
+    @Id
+    @Column(name = "donation_id")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name="calloutneed_id")
+    private CalloutNeed calloutNeed;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name="donor_id")
+    private Donor donor;
+
+    @Column(name = "donation_date")
+    private Date date;
+
+    @Column(name = "donation_quantity")
+    private int quantity;
+
+    public long getId() {

```

Comment:
Should this be returning a `Long`? I know it'll auto-unbox the `Long` to `long` but I'm not sure what the caller is going to expect. My gut tells me it should return a `Long`.

Reply:
good call I changed this!